testing/endtoend/components/eth1/node.go

```diff
@@ -127,12 +127,16 @@ func (node *Node) Start(ctx context.Context) error {
 		if err = runCmd.Start(); err != nil {
 			return fmt.Errorf("failed to start eth1 chain: %w", err)
 		}
-		if err = helpers.WaitForTextInFile(errLog, "Started P2P networking"); err != nil {
+		// TODO: the log is not very descriptive but it's always the first log where the chain has
+		// - a peer
+		// - http server started
+		// - genesis synced
+		if err = helpers.WaitForTextInFile(errLog, "Node revalidated"); err != nil {
```

Comment:
This log is sort of akward, looking for suggestions on this. adding the check for p2p start or peer found is still too early starting the eth1 client, but using Post Merge check saying it requires a beacon node is too late and the evaluator fails due to missing slots 1 and 2